README.md

```diff
@@ -1,101 +1,92 @@
-# HTML & CSS3 & JavaScript Course
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-If you are not familiar with linters, read [root level README](../README.md).
+# FoodAPI Capstone Project
 
-## Set-up GitHub Actions
+> This is a FoodAPI application.
 
-Please do the following **steps in this order**:
 
-1. In the first commit of your feature branch create a `.github/workflows` folder and add a copy of [`.github/workflows/linters.yml`](.github/workflows/linters.yml) to that folder.
-   - **Remember** to use the file linked above
-   - **Remember** that `.github` folder starts with a dot.
-2. **Do not make any changes in config files - they represent style guidelines that you share with your team - which is a group of all Microverse students.**
-   - If you think that change is necessary - open a [Pull Request in this repository](../README.md#contributing) and let your code reviewer know about it.
-3. When you open your first pull request you should see the result of the GitHub actions checks:
+## Built With
 
-![gh actions checks](../assets/images/gh-actions-html-css-checks.png)
+- **Languages:**
+  - HTML5 & CSS3
+  - JavaScript
 
-Click on the `Details` link of each action to see the full output and the errors that need to be fixed:
+- **Preprocessors:**
+  - Sass
 
-![gh actions failing checks](../assets/images/gh-actions-html-css-failing-checks.png)
+- **Technologies:**
+  - Webpack
 
-## Set-up linters in your local env
+- **Conventions and Methodologies:**
+  - Atomic Design
+  - BEM
 
-**Note**: The `npm` package manager is going to create a `node_modules` directory to install all of your dependencies. You shouldn't commit that directory. To avoid that, you can create a [`.gitignore`](https://git-scm.com/docs/gitignore) file and add `node_modules` to it:
+## Live Demo
+Click on this link to see this project live demo: [Live Demo Link](https://masangana.github.io/capstone-project/)
```

Comment:
- [ ] Your app is looking great! It would be great to get this link working to see your app live:

![Captura de pantalla 2022-06-24 a la(s) 1 19 37 p  m](https://user-images.githubusercontent.com/83717285/175629752-b05d0f69-df04-4d85-aa70-2e84c2e14ede.png)

Reply:
Done